identity-composer.html

```diff
@@ -1,70 +1,56 @@
-<div>
-  <h3>A credential consumer is requesting the following information:</h3>
-  <p>
-    Click each item and select a credential you would like to use
-    to fulfill the request.
-  </p>
-  <div class="btn-group" role="group">
-    <button ng-repeat="(property, choice) in choices"
-      type="button" class="btn btn-default"
-      ng-click="showChoice(property)">
-      <span ng-if="!choice.selected"
-        class="fa fa-question-circle"></span>
-      <span ng-if="choice.selected"
-        class="fa fa-check-circle text-success"></span>
-      {{choice.label}}<span ng-if="choice.optional"> (optional)</span>
-    </button>
-    <button
-      type="button" class="btn btn-default"
-      ng-disabled="!composed"
-      ng-click="done()">Done</button>
+<div ng-show="page == 'edit'" class="row row-centered edit-header">
+  <div class="col-xs-2">
+    <a class="text-center" ng-click="back()"><span style="margin-top: 25px;" class="glyphicon glyphicon-chevron-left"/>Back</a>
   </div>
-  <div ng-repeat="(property, choice) in choices track by property">
-    <div ng-show="choice.show">
-      <h4>Select a credential for <strong>{{choice.label}}</strong>:</h4>
-      <ul class="list-unstyled">
-        <li class="br-selectable well"
-          ng-class="{'br-selected': choice.selected === option.credential}"
-          ng-repeat="option in choice.options track by $index"
-          ng-click="choice.selected = option.credential">
-          <!-- TODO: replace with br-credential-thumbnail -->
-          <div class="section">
-            <h3 class="headline" style="margin-top: 5px">
-              {{option.credential.name || 'Credential'}}
-              <span class="btn-group pull-right">
-                <button type="button"
-                  class="btn btn-default"
-                  ng-click="$event.stopPropagation(); showCredential(option)">
-                  <i class="fa fa-trophy fa-lg"></i>
-                </button>
-              </span>
-            </h3>
-          </div>
-          <br-credential
-            br-credential="option.credential"
-            br-show-actions="false"></br-credential>
-        </li>
-        <li class="well"
-          ng-if="choice.options.length == 0">
-          <strong>No credentials found.</strong>
-        </li>
-      </ul>
+  <div class="col-xs-8">
+    <h2 class="text-center">Edit Selection</h2>
+  </div>
+</div>
+<div ng-show="page == 'front'" class="row text-center front-page">
```

Comment:
Testing the two out, it looks like ng-show performs decently better (it's especially noticeable because we do an animation between the front and edit pages). The animation docs also use ng-show when triggering animations, so maybe we should stick with it unless we notice especially slow page loads.